idunn/blocks/__init__.py

```diff
@@ -1,16 +1,32 @@
 from .opening_hour import OpeningHourBlock
+from .happy_hour import HappyHourBlock
 from .phone import PhoneBlock
 from .information import InformationBlock
 from .website import WebSiteBlock
 from .contact import ContactBlock
-from .wikipedia import WikipediaBlock, WikiUndefinedException, GET_WIKI_INFO, WikipediaCache
+from .wikipedia import (
+    WikipediaBlock,
+    WikiUndefinedException,
+    GET_WIKI_INFO,
+    WikipediaCache,
+)
 from .images import ImagesBlock
-from .services_and_information import ServicesAndInformationBlock, AccessibilityBlock, InternetAccessBlock, BreweryBlock, CuisineBlock
+from .services_and_information import (
+    ServicesAndInformationBlock,
+    AccessibilityBlock,
+    InternetAccessBlock,
+    BreweryBlock,
+    CuisineBlock,
+)
 from .grades import GradesBlock
-from .events import OpeningDayEvent, DescriptionEvent
+from .events import (
+    OpeningDayEvent,
+    DescriptionEvent,
+)
 
 ALL_BLOCKS = [
     OpeningHourBlock,
+    HappyHourBlock,
```

Comment:
this block also needs to be defined in `BLOCKS_BY_VERBOSITY` object (in utils.py), to be included in the "long" verbosity output.